Makefile

```diff
@@ -121,7 +121,21 @@ e2e-test: ## Run e2e tests against cluster pointed to by ~/.kube/config
 		-count 1 \
 		-timeout 90m \
 		-v \
-		./suites/... \
+		./suites/integration/... \
+		--ginkgo.focus="${FOCUS}" \
+		--ginkgo.timeout=90m \
+		--ginkgo.v
```

Comment:
are these same flags from above?
		-count 1 \
		-timeout 90m \

Reply:
Will double check with only the above flags and remove the gingko ones if possible.